src/header/header.scss

```diff
@@ -32,6 +32,13 @@
   height: 100%;
 }
 
+@mixin iui-page-header-left {
```

Comment:
Just applying this to whole header is not enough?

Reply:
`box-sizing`, `margin-bottom`, and `padding-bottom` could be applied to all zones (left, center, right) but if we apply `overflow` on the right zone we get a result that is different from what is currently live.

Correct on the top, incorrect on the bottom.
![Screen Shot 2022-05-24 at 2 08 47 PM](https://user-images.githubusercontent.com/849817/170103326-64a30211-76eb-4130-82f7-86ba8db52216.png)

Reply:
Ok, so this is possible:

- Use `:before` to place the 2px bottom border.
- `overflow` still needs to be applied on `.iui-page-header-left`, `.iui-header-breadcrumbs`, & `.iui-header-breadcrumbs-list`.  We could apply `overflow` on `@mixin iui-breadcrumbs` & `@mixin iui-breadcrumbs-list` instead of the header specific ones if we'd like.
- Inset the `box-shadow` of the active breadcrumb button.

However this does re-introduce this issue of text & icons appearing off center.
![Screen Shot 2022-05-19 at 3 51 40 PM](https://user-images.githubusercontent.com/849817/170326892-77f6989d-d67d-4ebe-8b13-23680e66bdea.png)

That could be fixed by giving ALL buttons within the header `padding-bottom: 2px`.  I'm not sure this is any better than what we have currently.